src/notes/FluxNotesEditor.scss

```diff
@@ -108,10 +108,20 @@
             &-inserter {
                 color: $state;
                 display: inline;
+
+                &-incomplete {
+                    color: $state;
+                    display: inline;
+                    border-bottom: 1.5px dashed $missing-data-red;
+                }
             }
         
             &-creator {
-                border-bottom: 1px solid $shr-context-line;
+                border-bottom: 1px dashed $shr-context-line;
```

Comment:
[MG4] When we rearranged the sass we lost consistency in the border thickness so this should be 1.5px instead of 1px